libvast/src/system/application.cpp

```diff
@@ -487,6 +487,9 @@ auto make_root_command(std::string_view path) {
   auto ob
     = opts("?vast")
         .add<std::string>("config", "path to a configuration file")
+        .add<bool>("bare-mode", "disable user and system configuration, schema "
+                                "and plugin  directories lookup (this may only "
+                                "be used on the command line)")
```

Comment:
Do we even need to add this here, since its not handled by caf anyways? Anyways there's a double space in "plugin  directories", and the option does not disable user and system configuration.

Reply:
Yes, otherwise it doesn't show up in `vast help`.